sidebars.js

```diff
@@ -50,17 +50,12 @@ const sidebars = {
     { type: 'link', label: 'NeonFaucet', href: 'https://neonfaucet.org/' },
     { type: 'link', label: 'NeonAnalytics', href: 'https://neon.aleph.cloud/' },
     { type: 'html', value: '<div class="sidebar-menu-divider"/>' },
-    { type: 'html', value: '<h4 class="sidebar-menu-title">Developers</h4>' },
+    { type: 'html', value: '<h4 class="sidebar-menu-title">Develop</h4>' },
     {
       type: 'doc',
       label: 'Connect to Neon RPC',
       id: 'developing/connect_rpc'
     },
-    {
```

Comment:
pls return back “Request tokens” section to this level, because it is two most asked by developers scenarios: 

1. Connect to RPC
2. Get test tokens

We did several interviews, asked developers for it and put it here